templates/zerver/help/require-topics.md

```diff
@@ -6,20 +6,20 @@ By default, users are not required to specify a topic in stream messages; if
 a user sends a message without a topic, the message's topic is displayed as
 **(no topic)**.
 
-If [message editing](/help/configure-message-editing-and-deletion) is
-enabled, any other user can then edit the topic of messages without a
-topic to set a topic, regardless of the value of the [topic editing
-policy](/help/configure-who-can-edit-topics).
+If [message editing](/help/configure-message-editing-and-deletion)
+is enabled, then any user can [add a topic](/help/rename-a-topic)
+to messages without a topic, regardless of the value of the
+[topic editing policy](/help/configure-who-can-edit-topics).
 
```

Comment:
How about something like this to make this whole page easier to read?

---


Administrators can configure whether each message must have a specified topic.

[instructions go here]

## Messages without a topic

If a user sends a message without a topic, the message's topic is displayed as
**(no topic)**.

When [message editing](/help/configure-message-editing-and-deletion)
is enabled, any user can [add a topic](/help/rename-a-topic)
to messages without a topic. They can do so regardless of whether they can
[edit topics in general](/help/configure-who-can-edit-topics).

Reply:
That makes sense. Do we want to leave the admin note at the top or move it to be above the instructions tab block?